packages/codemirror/src/mimetype.ts

```diff
@@ -51,6 +51,9 @@ class CodeMirrorMimeTypeService implements IEditorMimeTypeService {
    * If a mime type cannot be found returns the defaul mime type `text/plain`, never `null`.
    */
   getMimeTypeByFilePath(path: string): string {
+    if (PathExt.extname === 'ipy') {
```

Comment:
Ahh sorry, that was silly! Was staring at it for the last hour and overthinking it, thanks!!